tests/performance_tests/log_reader.py

```diff
@@ -237,11 +237,11 @@ def scrapeTrxGenLog(trxSent, path):
     with selectedopen(path, 'rt') as f:
         trxSent.update(dict([(x[0], x[1]) for x in (line.rstrip('\n').split(',') for line in f)]))
 
-def scrapeBlockTrxDataLog(trxDict, path):
+def scrapeBlockTrxDataLog(trxDict, path, nodeosVersion=None):
     #blockTrxData.txt
     selectedopen = selectedOpen(path)
     with selectedopen(path, 'rt') as f:
-        if Utils.getNodeosVersion().split('.')[0] == "v2":
+        if Utils.getNodeosVersion().split('.')[0] == "v2" or nodeosVersion == "v2":
```

Comment:
Should just use `nodeosVersion` here and remove the `Utils.getNodeosVersion()` call.

Might be worth looking at our use of `Utils.getNodeosVersion()` throughout performance harness and see if we can limit this to a single place and then always get it out of `ClusterConfig`, for example.  Makes it easier to update in the future if we move away from `Utils.getNodeosVersion()` as the way we get nodeos version.

Reply:
Thanks.